sktime/datatypes/_panel/_check.py

```diff
@@ -1184,7 +1184,46 @@ def _check_polars_panel(obj, return_metadata=False, var_name="obj", scitype="Pan
 
 
 class PanelGluontsList(ScitypePanel):
-    """Data type: polars.DataFrame based specification of panel of time series.
+    """Data type: gluonTS representation of univariate and multivariate time series.
+
+    Name: ``"gluonts_ListDataset_panel"``
+
+    Short description:
+
+    A ``gluonts.dataset.common.ListDataset``, a ``list`` of ``dict``,
```

Comment:
this is confusing - what is the exact type now? A `ListDataset` or a `list`?

Surely this is not the same python type.

Reply:
The exact type is ``list``. But the object format itself replicates the ``gluonTS`` library's ``ListDataset``. See below.

https://ts.gluon.ai/v0.11.x/api/gluonts/gluonts.dataset.common.html#gluonts.dataset.common.ListDataset

I wrote that in, since I felt I should refer to the original specs from ``gluonTS``. But if you think it's confusing, I will remove it.

Reply:
oh, I see - the constructor is called `ListDataset`, but it actually produces a list. That works, I was just confused about the specification as I have never used `gluonts` directly.